### PR TITLE
Add blur variants for context-tags

### LINKED ISSUE #15588: Add missing blur variants for context tags
# Problem

Current implementation of context tags lack of their blur variants (both dark and light)

Context tags:
image
[Designs](https://www.figma.com/file/WQZcp6S0EnzxdTL4taoKDv/Design-System-for-Mobile?node-id=1336-34320&t=gKioduBuaEid7HSi-4)

## Acceptance Criteria

Context tags can be displayed in their blur variants.

## Future Steps

The issue #15480 can be solved

### PR DESCRIPTION
**Pull request by ulisesmac**
[comment]: # (Please replace ... with your information. Remove < and >)
[comment]: # (To auto-close issue on merge, please insert the related issue number after # i.e fixes #566)

fixes #15588 

### Summary

[comment]: # (Summarise the problem and how the pull request solves it)
This PR adds a blur variants for context-tags and performs a small refactoring.

image
image

#### Platforms

- Android
- iOS

### Steps to test

- Open Status
- Navigate to Quo2 Preview -> tags -> context-tags
- Test the blur preview

status: ready

### CODE REVIEW

File: src/quo2/components/tags/context_tag/style.cljs [hunk 1]
```diff
@@ -0,0 +1,55 @@
+(ns quo2.components.tags.context-tag.style
```
Comment (ulisesmac) [inline src/quo2/components/tags/context_tag/style.cljs:1]: Created this new namespace to split the styles into a separate file.

File: src/quo2/components/tags/context_tag/view.cljs [hunk 2]
```diff
@@ -0,0 +1,85 @@
+(ns quo2.components.tags.context-tag.view
```
Comment (ulisesmac) [inline src/quo2/components/tags/context_tag/view.cljs:1]: Moved the previos code from `quo2.components.tags.context-tags` -> `quo2.components.tags.context-tag.view`

File: src/status_im2/contexts/quo_preview/tags/context_tags.cljs [hunk 3]
```diff
@@ -37,7 +37,10 @@
               {:key   :audio
                :value "Audio"}
               {:key   :community
-               :value "Community"}]}])
+               :value "Community"}]}
+   {:label "Blur"
```
Comment (ulisesmac) [inline src/status_im2/contexts/quo_preview/tags/context_tags.cljs:41]: Added a `:blur?` option to descriptors in preview screen

File: src/status_im2/contexts/quo_preview/tags/context_tags.cljs [hunk 4]
```diff
@@ -51,73 +54,113 @@
...
                                          :value (get-in contacts [public-key :primary-name])})
                                       (vals contacts))
-            current-username (if (seq (:contact @state))
-                               (->> @state
-                                    :contact
-                                    contacts
-                                    multiaccounts/displayed-name)
-                               "Please select a user")
-            descriptor
-            (cond
```
Comment (ulisesmac) [inline src/status_im2/contexts/quo_preview/tags/context_tags.cljs:73]: In the preview screen, the descriptor existed only for some types of context-tags, if we chose a type not contemplated, the descriptor menu disappeared making impossible to switch back to another type, now this is fixed

**Review by smohamedjavid — APPROVED** [review]
Thanks, @ulisesmac! for splitting the **view** and **style** of  the `context-tag` component 🚀

**Pull request merged**